tests/integration/query/one_to_many/with_filter_test.go

```diff
@@ -17,274 +17,468 @@ import (
 )
 
 func TestQueryOneToManyWithNumericGreaterThanFilterOnParent(t *testing.T) {
-	test := testUtils.RequestTestCase{
+	test := testUtils.TestCase{
```

Comment:
praise: Thank you for taking the time to convert these to the newer framework :)